paasta_tools/utils.py

```diff
@@ -2757,7 +2757,10 @@ def prompt_pick_one(sequence: Collection[str], choosing: str) -> str:
         sys.exit(1)
 
     global_actions = [str('quit')]
-    choices = [(str(item), str(item)) for item in sequence]
```

Comment:
Any idea why we `str()`'d this originally? Does the choice library tolerate "Nones" and other nonsense?

Reply:
I'm not sure, but now that we're using mypy we know sequence is only going to contain strings so it's fine.

Also @solarkennedy I don't have write access any more so can you merge this and/or re-add me? Thanks :)